docs/monitoring.md

```diff
@@ -0,0 +1,294 @@
+   ```
+
+CONFIG JSON
+
+   ```bash
+   {
+    "model_config_list": [
+        {
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true
+            }
+        }
+   }
+   ```
+
+## Change the default list of metrics
+
+You can enable from one up to all the metrics available at once.
+
+To enable specific set of metrics you need to specify the metrics_list flag or json setting:
+
+CLI
+
+   ```bash
+         docker run --rm -d -p 9000:9000 -p 8000:8000 openvino/model_server:latest \
+               --model_name resnet --model_path gs://ovms-public-eu/resnet50  --port 9000 \
+               --rest_port 8000 \
+               --metrics_enabled \
+               --metrics_list ovms_requests_success,ovms_infer_req_queue_size
+   ```
+
+CONFIG CMD
+
+   ```bash
+         docker run --rm -d -v -d -v ${PWD}/workspace:/workspace openvino/model_server \
+            --config_path /workspace/config.json -p 9000:9000 -p 8000:8000 openvino/model_server:latest \
+            --rest_port 8000 \
+            --port 9000
+   ```
+
+CONFIG JSON
+
+   ```bash
+   {
+    "model_config_list": [
+        {
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true,
+                "metrics_list": ["ovms_requests_success", "ovms_infer_req_queue_size"]
+            }
+        }
+    }
+    ```
+
+CONFIG JSON WITH ALL METRICS ENABLED
+
+```bash
+   {
+    "model_config_list": [
+        {
+           "config": {
+                "name": "resnet",
+                "base_path": "/workspace/resnet-50-tf",
+                "layout": "NHWC:NCHW",
+                "shape": "(1,224,224,3)"
+           }
+        }
+    ],
+    "monitoring":
+        {
+            "metrics":
+            {
+                "enable" : true,
+                "metrics_list": 
+                    [ "ovms_requests_success",
+                    "ovms_requests_fail",
+                    "ovms_inference_time_us",
+                    "ovms_wait_for_infer_req_time_us",
+                    "ovms_request_time_us",
+                    "ovms_current_requests",
+                    "ovms_infer_req_active",
+                    "ovms_streams",
+                    "ovms_infer_req_queue_size"]
+            }
+        }
+    }
+    ```
+
+## Example response from metrics endpoint
+
+To use data from metrics endpoint you can use the curl command:
+```bash
+    curl http://localhost:8000/metrics
+```
+
+The example output looks like this:
```

Comment:
maybe instead of using a part of the output it will be better to use a link to a complete response (including values after running some load)

Reply:
Ok.